internal/commands/document.go

```diff
@@ -18,14 +18,17 @@ type Header struct {
 	Title       string
 	Description string
 	Resources   string
+	Anchor      string
 }
 
 // Document is a single policy document.
 type Document struct {
-	Header   Header
-	Severity string
-	URL      string
-	Rego     string
+	Header       Header
+	Severities   string
+	HasViolation bool
```

Comment:
opinion: I would approach this as having a type that represents the overall document that we render and then have that type include slices of Policies. Something like..

```go
type Document struct {
  Violations []Policy
  Warning []Policy
  InsertOtherPolicyTypeOrSeverityHere []Policy
}
```

```go
type Policy struct {
  Header string
  URL string
  ...
}
```

This gets us away from boolean flags, and represents more closely what is being rendered on the page. What is here currently works perfectly fine, but just wanted to present another view.

Reply:
I'm not sure I agree with this. If we split the policies into separate slices, the template then has to have a `{{range .<<Severity>>}}` iterator for each which makes the template longer, ensures it has duplicated sections (opposite of DRY), and doesn't automatically handle new severities. 

While it would add a bit more code, we could add a `AllSeverities []string` to the `Documentation` struct and use that to determine what to write out in a severity agnostic manner. However, as I mentioned in my other comment, this tool is targeting a very specific use case so we may not need that level of flexibility.

I think I will go with your suggestion for now, but if this tool ever significantly expands its scope this will need to be revisited.

Reply:
If there's not enough value add and you're on the fence, I wouldn't worry about it. It's very possible the flexibility isn't needed.

To fix the DRY issue, it should be possible to do something like a map[string][]Policy where the map key is the severity. Then range on the single map.